vice/src/resid/Makefile.am

```diff
@@ -2,9 +2,19 @@
 
 AR = @AR@
 
+AM_CFLAGS = @VICE_CFLAGS@
+
+#AM_CXXFLAGS = @VICE_CXXFLAGS@
+
 noinst_LIBRARIES = libresid.a
 
-libresid_a_SOURCES = sid.cc voice.cc wave.cc envelope.cc filter.cc dac.cc extfilt.cc pot.cc version.cc
+if NEW_8580_FILTER
```

Comment:
Should this be defined or not? Remember Retroarch core is not using configure. We are using a makefile. Did you edit Makefile.libretro?

Reply:
None of your changes to Makefile.am etc. will make any difference. We are not using configure in Retroarch. We are using Makefile.libretro etc.

Reply:
Remember Retroarch core is not using configure. We are using Makefile.libretro and Makefile.common.

Did you test that no change to Makefile.common is needed for example?